apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -498,45 +536,39 @@ function initializeBlocklyWrapper(blocklyInstance) {
   };
 
   // Used in levels with pre-defined "Blockly Variables"
-  blocklyWrapper.WorkspaceSvg.prototype.registerGlobalVariables = function (
-    variableList
-  ) {
+  extendedWorkspaceSvg.registerGlobalVariables = function (variableList) {
     this.globalVariables = variableList;
     this.getVariableMap().addVariables(variableList);
   };
 
-  blocklyWrapper.WorkspaceSvg.prototype.getContainer = function () {
+  extendedWorkspaceSvg.getContainer = function () {
     return this.svgGroup_.parentNode;
   };
 
-  const googleBlocklyBlocklyResize =
-    blocklyWrapper.WorkspaceSvg.prototype.resize;
-  blocklyWrapper.WorkspaceSvg.prototype.resize = function () {
-    googleBlocklyBlocklyResize.call(this);
-    if (cdoUtils.getToolboxType() === ToolboxType.UNCATEGORIZED) {
-      this.flyout_?.resize();
-    }
-  };
-
-  blocklyWrapper.WorkspaceSvg.prototype.events = {
+  extendedWorkspaceSvg.events = {
     dispatchEvent: () => {}, // TODO
   };
 
   // TODO - called by StudioApp, not sure whether they're still needed.
-  blocklyWrapper.WorkspaceSvg.prototype.setEnableToolbox = function (
-    enabled
-  ) {};
-  blocklyWrapper.WorkspaceSvg.prototype.traceOn = function (armed) {};
+  extendedWorkspaceSvg.setEnableToolbox = function () {};
+  extendedWorkspaceSvg.traceOn = function () {};
+
+  const extendedVariableMap = blocklyWrapper.VariableMap
+    .prototype as ExtendedVariableMap;
 
-  blocklyWrapper.VariableMap.prototype.addVariables = function (variableList) {
+  extendedVariableMap.addVariables = function (variableList) {
     variableList.forEach(varName => this.createVariable(varName));
   };
 
-  // TODO - used for spritelab behavior blocks
-  blocklyWrapper.Block.createProcedureDefinitionBlock = function (config) {};
+  // Used for spritelab behavior blocks.
+  // We can remove this once we are ready to no longer support sprite lab on CDO Blockly.
```

Comment:
I'm seeing this called once in `customBlocks/cdoBlockly` and one in a Dance file for behavior definition blocks. We don't actually have behavior definition blocks in Dance, so we could probably clean this up quickly as a follow up. I don't think that we need to keep it here just for Sprite Lab on CDO Blockly because this wrapper isn't used in that circumstance any way.

Reply:
added this to the follow up ticket!